### ISSUE TITLE
L2 loss channel-wise missing?

### ISSUE THREAD

**Issue by Franborat** [Ha0Tang/GestureGAN#4]
Hello!

I've been reading the GestureGAN paper and the way in which calculating the L2 loss in the generator channel-wise avoids "channel pollution". However, I can not find it implemented in the code. Only the L1 loss is calculated channel-wise, which the paper states that is not necessary.  Is this an error en in the code, and when calculating L1 loss it was meant to calculate L2?

Thanks!

**Comment by Ha0Tang** [comment]
Thanks for your interest in our work.
1. You can replace `self.criterionL1` with `self.criterionL2` to obtain the channel-wise L2 loss, where `self.criterionL2 = torch.nn.MSELoss()`.
2. We also found that in our experiments, the channel-wise L1 loss achieves better results than the traditional L1 loss, although, in theory, both should be the same.

**Comment by Franborat** [comment]
All right! Thank you for the answer ! 
I'll try both and see what works best in my research 👍
